Dockerfile

```diff
@@ -62,43 +61,40 @@ FROM base AS dependency_builder
 
 COPY acs/ /acs
 
-RUN yum -y install  \
-	curl \
-	git-lfs \
-	ksh \
-	mc \
-	nc \
-	patch \
-        # Needed by buildJacOrb
-	rsync \
-	screen \
-	subversion \
-	tree \
-	unzip \
-	vim \
-	wget \
-	xterm && \
-	cd /acs/ExtProd/PRODUCTS && \
-    ## Get missing (super old) libraries
-    wget https://sourceforge.net/projects/gnuplot-py/files/Gnuplot-py/1.8/gnuplot-py-1.8.tar.gz/download -O gnuplot-py-1.8.tar.gz && \
-    wget https://sourceforge.net/projects/pychecker/files/pychecker/0.8.17/pychecker-0.8.17.tar.gz/download -O pychecker-0.8.17.tar.gz && \
-    wget https://sourceforge.net/projects/numpy/files/OldFiles/1.3.3/numarray-1.3.3.tar.gz && \
```

Comment:
So you removed these .. I think that is super .. but can you comment a bit on how you found out, that it is okay to remove them? Did you test anything in particular? Or did you discuss this with anybody?

I have no idea which part of ALMA/ACS might need this.

Reply:
I have no idea either. I've just tried to remove them and the build process still continue working 😂

It would be great if @ACS-Community/developers can give a hand on this.

Reply:
@tstaig said:

> Sorry, I'll reply to the list, but no, they're not used by ACS itself, they may be used by some specific projects, but that's also unlikely
> 2:22
> As matter of fact they should no longer be part of the installation script for a while now:
> 
> ```
> commit 7646c821e41808468db7e305ef01355f16e1aff4
> Author: Tomas Staig <tomas.staig@alma.cl>
> Date:   Thu Feb 20 08:48:32 2020 -0300
>     ICT-16349: Removing gnuplot, numarray and pychecker from buildPyModules as they're provided elsewhere
>     Former-commit-id: 5b2a7f26269e3e036335afea91b3a26755f23aaf
> ``` 
> Here, elsewhere means somewhere else in the ALMA repo besides ACS.

Reply:
Great!